### ISSUE TITLE
Broken tree view of Code Browser 0.18.0

### ISSUE THREAD

**Issue by kmoski** [TabbyML/tabby#3314]
File system repo tree:
```
$ tree
.
├── code0.c
└── dir0
    ├── code1.c
    └── dir1
        ├── code2.c
        └── dir2
            └── code3.c

4 directories, 4 files
```
Web Code Browser tree:
![Peek 2024-10-24 14-45](https://github.com/user-attachments/assets/d9bf35e5-a062-4932-8966-82db0901c8bb)

**Comment by wsxiaoys** [comment]
Hi @kmoski - could you share your test directory as an github repository so we can try reproduce the issue, thank you!

**Comment by kmoski** [comment]
https://github.com/kmoski/tabby-issue-3314

![Peek 2024-10-24 15-27](https://github.com/user-attachments/assets/efd707b7-7bae-426d-9648-c4479721fab0)

**Comment by icycodes** [comment]
This bug has been fixed and is included in the v0.22.0 release.
